back/src/dals/room/repository/room.mock.ts

```diff
@@ -1,5 +1,5 @@
 import produce from 'immer';
-import { InputMessageTypes } from 'messages';
+import { InputMessageTypes } from 'pods/messages';
```

Comment:
I think this import is not necessary, isn't it?

Reply:
Which one do you mean? All the imports in that file are in use

Reply:
I mean we don't need `import { InputMessageTypes } from 'pods/messages';` in dals repository, isn't it?

Reply:
Sorry, but I am lost, it is being using here:

 switch (action) {
    case InputMessageTypes.TRAINER_APPEND_TEXT:
      newContent += '\n' + newRoomInfo.content;
      break;
    case InputMessageTypes.TRAINER_SET_FULL_TEXT:
      newContent = newRoomInfo.content;
      break;
  };

Reply:
I have just pushed some changes to solve the problem.

Hopefully it works for you.